rise-financial-es6.js

```diff
@@ -336,6 +350,11 @@
         }
       }
 
+      connectedRef = firebase.database().ref( ".info/connected" );
+      connectedRef.on( "value", ( function onConnectionStateChanged( snap ) {
+        this.firebaseConnected = snap.val();
```

Comment:
@settinghead I think `firebaseConnected` should be added to "private" vars here - https://goo.gl/9oj1Fl and default to `true`. Also can you use the usual _ prefix,  `this._firebaseConnected`.